db/migrations/20240604184524_PageSnapshot.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE PageSnapshot (
+    page_snapshot_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    page_id UUID REFERENCES Pages (page_id) NOT NULL,
+    raw_html TEXT NOT NULL,
+    fetched_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+    -- title TEXT NOT NULL,
+    -- cleaned_html TEXT NOT NULL,
+    -- cleaned_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+    -- markdown_id UUID
```

Comment:
commented out these properties for now because they weren't being used. I'm planning to use them in the next features

Reply:
Nice! Fine to leave them here for reference if you want

But when we go to actually add these we'll want to do it in new migrations instead of editing older ones.

For migrations once they land on the 'main' branch they really should be 'frozen' and never changed! This helps avoid issues like you've seen where a previous migration is edited and SQLx goes crazy until you delete your DB and create it again

So when we want to add these columns later we will do that with NEW migrations and then some `ALTER TABLE` sql commands to add columns. And we'll have to chat about what to do about columns we want to add that are not nullable, but lets cover that when we get there 😆

Reply:
Okay awesome! I will reach out when I get started on this! Thank you for pointing it out for the future!